src/peft/peft_model.py

```diff
@@ -461,6 +494,81 @@ def set_adapter(self, adapter_name):
     def active_peft_config(self):
         return self.peft_config[self.active_adapter]
 
+    def push_to_hub(
```

Comment:
As https://github.com/huggingface/transformers/pull/24074 will be merged maybe we should remove this method? @pacman100

Reply:
Or let's keep it until the next `transformers` release (soon)